### PR TITLE
added options chart data endpoint & function to retrieve bars

### PR DESCRIPTION
**Pull request by boonyeow**
To get a list of options available for a stock, you can simply invoke the `option_chain` function. 
You can retrieve `derivativeIds` for specific options by calling e.g. `option_chain[0]["put"]["tickerId"]`

**data provided is not real-time and has a 15 min delay

### CODE REVIEW

**Comment by hfacey** [comment]
Thanks for coding this up! 
FYI, if you're subscribed to the OPRA realtime feed and authenticated when making the call, there shouldn't be  a delay. And the response includes a delay field, if it is delayed

File: webull/webull.py [hunk 1]
```diff
@@ -1046,6 +1045,39 @@ def get_bars(self, stock=None, tId = None, interval='m1', count=1, extendTrading
             df.loc[to_datetime(datetime.fromtimestamp(int(row[0])).astimezone(time_zone))] = data
         return df.iloc[::-1]
 
+    def get_options_bars(self, derivativeId=None, interval='1m', count=1, timeStamp=None):
+        '''
+        get bars returns a pandas dataframe
+        params:
+            interval:1m, 5m, 30m, 60m, 1d
+            count: number of bars to return
+            timeStamp: If epoc timestamp is provided, return bar count up to timestamp. If not set default to current time.
```
Comment (hfacey) [inline webull/webull.py:1054]: you've neglected to supply the "direction" parameter. I found that if that parameter is missing or set to anything except "1", the time stamp is ignored and all data for for the derivate (up to count) is returned

**Comment by tedchou12** [comment]
Thank you! Looks good. Let me test tmr, I will merge it!

File: webull/webull.py (hunk 1 above)
Comment (boonyeow) [reply webull/webull.py:1062]: Yep, you're right! I've made the changes, thanks for pointing that out!
I also found that if the direction is set to 1, it ignores the `count` parameter and returns all bars on and after the timestamp

**Pull request merged**